setup/module/api/core.tf

```diff
@@ -104,3 +104,31 @@ resource "aws_iam_group_policy" "mod" {
   group  = "${aws_iam_group.mod.id}"
   policy = "${element(data.template_file.group_policy.*.rendered, count.index)}"
 }
+
+data "aws_ami" "linux" {
+  most_recent = true
+  executable_users = ["self"]
```

Comment:
this this necessary? https://www.terraform.io/docs/providers/aws/d/ami.html

Reply:
I don't think it's necessary, but as the docs say:
"Limit search to users with explicit launch permission on the image."

It's an untested use case, but in the event that an image got published that we didn't have launch permissions on, this _could_ come in handy.

Reply:
Ah I see. I'm not 100% what "explicit launch permission" entails, but I'm worried about the inverse problem: no one has "explicit launch permission" on AMIs w/o going in and setting some permission yourself. Anyways, this just needs to be tested.

Reply:
I removed this param for now as the matter of permissions for images should be out of ours hands, otherwise it somewhat defeats the purpose of this PR.